src/styles/colors.js

```diff
@@ -34,6 +34,9 @@ export default {
     greenDefaultButtonPressed: '#467164',
     greenDefaultButtonDisabled: '#8BA69E',
 
+    // Picker option text color, we set this to avoid white on white dropdowns in Windows
```

Comment:
This comment could be clearer or perhaps moved top a more context appropriate location. "white on white dropdowns in Windows" didn't really paint a clear picture in my mind about what we are fixing or why setting this color would fix an issue. In this file - we're just defining colors and there are no "pickers".

Reply:
Good point, I'll just remove this comment since I already have one in `Picker` explaining why we need to decorate each item.